### ISSUE TITLE
deploy ros realsense driver to apt-get

### ISSUE THREAD

**Issue by SteveMacenski** [realsenseai/realsense-ros#370]
It would make it much easier to use this software if there was a apt-get package for it so I dont have to build from source along with my code

**Comment by RealSenseCustomerSupport** [comment]
[Please Ignore - RealSense system comment]

**Comment by RealSenseCustomerSupport** [comment]
----------------------------------------------

Close this issue for lack of activity. Please feel free to reopen if the issue is still relevant.

**Comment by SteveMacenski** [comment]
This is still revelent, its waiting on actions from @RealSenseCustomerSupport

**Comment by doronhi** [comment]
> This is still revelent, its waiting on actions from @RealSenseCustomerSupport

I agree.

**Comment by RealSenseCustomerSupport** [comment]
----------------------------------------------

Closing this issue since there has been no activity. Please feel free to let us know  if the feature is still needed. Thank you.

**Comment by SteveMacenski** [comment]
This feature is still needed and relevant. See above. Please stop closing issues without activity. Lack of activity is not a lack of issue, its a lack of interest in resolving it.

**Comment by RealSenseCustomerSupport** [comment]
----------------------------------------------

No problem, we will keep this issue open for the apt-get package of the RealSense ROS Wrapper.
Thank you.

**Comment by RealSenseCustomerSupport** [comment]
----------------------------------------------

Hi @SteveMacenski,
The latest wrapper 2.2.13 is available as ros-melodic-realsense2-camera and ros-kinetic-realsense2-camera.
https://github.com/IntelRealSense/realsense-ros#method-1-the-ros-distribution
Please give it a try. Thank you.

**Comment by SteveMacenski** [comment]
there's now a new distribution: Noetic to support as well

**Comment by RealSenseCustomerSupport** [comment]
----------------------------------------------

Hi @SteveMacenski, thank you for bring it to our attention about the upcoming (May 2020) ROS Noetic release.
The current realsense-ros wrapper still focus on supporting the LTS Melodic and Kinetic.

Please feel free to create a new feature request ticket for our development team to add the Noetic support. Thank you.

This case will be closed in 7 days unless otherwise. Thank you

**Comment by SteveMacenski** [comment]
I think that still follows in the scope of this ticket

**Comment by doronhi** [comment]
This issue's topic is "deploy ros realsense driver to apt-get" which we did so as far as I understand it, this issue can be closed now.
The request to support Noetic should be opened as a new issue. Naturally, now that we deployed the driver to apt-get, it can be assumed that support for Noetic will include that as well.
@SteveMacenski , It may seem like a technicality but it allows better tracking of the work being done.
